packages/cactus-cmd-api-server/src/main/json/openapi.json

```diff
@@ -74,6 +74,10 @@
             "PrometheusExporterMetricsResponse": {
                 "type": "string",
                 "nullable": false
+            },
+            "GetOpenApiSpecResponse": {
```

Comment:
@charellesandig Please include the major  version number in the model name so that once we do more major releases we can more easily maintain the different versions in parallel. Please see the other model names for the existing naming convention to mimic. (the ones where the convention does exist - unfortunately not all models)